public/js/presenter.js

```diff
@@ -278,7 +278,9 @@ function openNotes()
   if (mode.notes) {
     try {
       if(notesWindow == null || typeof(notesWindow) == 'undefined' || notesWindow.closed){
-          notesWindow = window.open('', '', 'width=350,height=450');
+          // yes, the explicit address is needed. Because Chrome.
```

Comment:
There was a time that this would have been "Because IE."  I love how the world turns